libs/datamodel/core/tests/config/sources.rs

```diff
@@ -135,12 +135,11 @@ fn must_error_for_empty_provider_arrays() {
     let error = super::parse_config(dml).map(drop).unwrap_err();
 
     let expectation = expect![[r#"
-        [1;91merror[0m: [1mError validating: This line is not a valid definition within a datasource.[0m
+        [1;91merror[0m: [1mError validating datasource `myds`: The provider argument in a datasource must be a string literal[0m
           [1;94m-->[0m  [4mschema.prisma:2[0m
         [1;94m   | [0m
         [1;94m 1 | [0mdatasource myds {
-        [1;94m 2 | [0m  [1;91mprovider = [][0m
-        [1;94m 3 | [0m  url = "postgres://"
+        [1;94m 2 | [0m  provider = [1;91m[][0m
```

Comment:
I'd consider this a better error message.

Reply:
The `url` line not appearing is intentional?

Reply:
Not intentional, but expected with this type of errors (they just have fewer context lines)